worlds/ed_queue_demo/ed_queue_demo/domain.py

```
@@ -0,0 +1,34 @@
+from tangl.story import Block
+
+
+class EdQueueSimulation(QueueSimulation):
+    """Deterministic ED queue configuration for the demo world."""
+
+    patient_arrivals: dict[str, int] = {
+        "P1": 0,
+        "P2": 2,
+        "P3": 4,
+        "P4": 8,
+    }
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)